test/runtests.jl

```diff
@@ -508,4 +508,13 @@ end
         @btime schreier_sims($(gens(G)))
         # GAP: 17 ms vs 50 ms here
     end
-end
\ No newline at end of file
+end
+
+@testset "Iterate over PrmGroup" begin
+	G = PermGroup(6);
+	K = PrmGroup(rand(G,2));
+	elements = [g for g in K]
+	@test unique!(elements) == elements
```

Comment:
this tests nothing;
use `uniq_elements = unique(elements); @test uniq_elements == elements`